blockly-ws-search/webpack.config.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * @fileoverview Webpack configuration file.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = env => {
+    return {
+        target: 'web',
+        mode: env.mode,
+        entry: './src/WorkspaceSearch.js',
+        devtool: 'source-map',
+        output: {
+            path: path.resolve(__dirname, 'dist'),
+            filename: 'workspace-search.umd.js',
+            libraryTarget: 'umd',
+            globalObject: 'this'
+        },
+        module: {
+            rules: [{
+                test: /\.js$/,
+                exclude: /(node_modules)/,
+                use: {
+                    loader: "babel-loader",
+                    options: {
+                        presets: ["@babel/preset-env"]
+                    }
+                }
+            }]
+        },
+        devServer: {
+            openPage: 'test',
```

Comment:
Configure the devServer port to 3000. Keeps it consistent with the rest of the repo. 
```
        port: 3000,
```

Reply:
done